### DISCUSSION TITLE
How to auto preview a ppt file after document ready?

### DISCUSSION THREAD

**Discussion by qomosoloto** [ONLYOFFICE/DocumentServer#2410] [Q&A]
Hi all, I want to preview the slides after the ducument ready. How to archive? I read through the onlyoffice docuemnts but got nothing. By the way, trying with `document.querySelector('the-preview-buttong-selector').click()` didn't work as expected either，the selector returned null!!

(PS: the editor is on `view mode` .)

**Comment by Rita-Bubnova** [comment]
@regent112, Take a look, please.
